DataFormats/FWLite/interface/Event.h

```diff
@@ -87,7 +88,8 @@ namespace fwlite {
       public:
          // NOTE: Does NOT take ownership so iFile must remain around
          // at least as long as Event
-         Event(TFile* iFile);
+         Event(TFile* iFile, bool useCache=true,
+               std::function<void (TBranch*)> baFoo=[](TBranch*){});
```

Comment:
What is the purpose of adding `baFoo`? Also, a better name is needed to describe the purpose.